modules/react/modal/stories/Modal.stories.mdx

```diff
@@ -81,6 +83,31 @@ requires a `label` prop.
 
 <ExampleCodeBlock code={CustomTarget} />
 
+### Body Content Overflow
+
+There are a few ways to handle body content overflowing. The easiest is to use `Modal.Card` as a
+vertical stack using `VStack` and setting the max height to the view height minus the desired margin
+around the card (default margin is `xl` or `40px`). The `VStack` is a flexbox container than
```

Comment:
`is a flexbox container than` -> `is a flexbox container that`

Reply:
Use the suggestion feature: 
<img width="30" alt="image" src="https://user-images.githubusercontent.com/338257/150197597-4f7f0b7f-17bb-4cc9-a514-cea4035ee1dc.png">

It makes it easy to make suggestions that are easy to commit